### PR TITLE
metrics: Add Cassandra README

### LINKED ISSUE #4857: add Cassandra README

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds a README for Cassandra metrics test.

Fixes #4857

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
@jodh-intel any idea of the static-check failure? I saw the logs but I could not find why is this failing? thanks

**Comment by gkurz** [comment]
> @jodh-intel any idea of the static-check failure? I saw the logs but I could not find why is this failing? thanks

@GabyCT isn't it https://github.com/kata-containers/tests/runs/6922105439?check_suite_focus=true#step:8:228 ?
```
ERROR: Document metrics/disk/cassandra_kubernetes/README.md is not referenced
```

i.e. the following requirement in `static-checks.sh` :
```
        # Every document in the repo (except a small handful of exceptions)
        # should be referenced by another document.
```

File: metrics/disk/cassandra_kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Kata Containers Cassandra Metrics
+
+Kata Containers provides a series of read and write performance tests using Cassandra Stress tool.
+The Cassandra Stress tool is a Java-based stress testing utility for basic benchmarking
+and load testing a cluster. This tool help us to populate a cluster and stress
+testing CQL tables and queries.
```
Comment (cmaf) [inline metrics/disk/cassandra_kubernetes/README.md:6]:
```suggestion
and load testing a cluster. This tool helps us to populate a cluster and stress
test CQL tables and queries. 
```

**Review by cmaf — COMMENTED** [review]
@GabyCT a couple syntax suggestions, otherwise LGTM

**Comment by GabyCT** [comment]
@gkurz thanks for the info :)

**Comment by GabyCT** [comment]
@cmaf changes applied

**Review by cmaf — APPROVED** [review]

**Review by gkurz — APPROVED** [review]
LGTM. Thanks @GabyCT !

**Comment by jodh-intel** [comment]
@gkurz - it was prolly https://github.com/kata-containers/tests/issues/4304. It would be great if someone could tal at that issue.

**Pull request merged**